user_story/__init__.py

```diff
@@ -1,4 +1,5 @@
-# coding: utf-8
-from . import model
+# -*- coding: utf-8 -*-
```

Comment:
The original one and this one works:

ref.

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms